src/main/java/lithium/university/TwitterRetrieve.java

```diff
@@ -15,15 +15,12 @@ public static void main(String[] args) {
         }
     }
 
-
-    public void retrieveFromTwitter(Twitter twitter, final int TWEET_TOTAL) throws TwitterException {
-        /*Fetch a titak.. I mean number of tweets*/
+    /*
+    * Gets the data from twitter and returns a list of the statuses
+    * */
+    public List<Status> retrieveFromTwitter(Twitter twitter, final int TWEET_TOTAL) throws TwitterException {
+        String timeline[] = new String[TWEET_TOTAL];
```

Comment:
is paging still needed?

Reply:
Paging allows you to specify the number of tweets to display from the home timeline. It's not *technically* needed if the user is fine with displaying the default 20 tweets